src/bijectors/corr.jl

```diff
@@ -61,26 +61,24 @@ Note: The implementation doesn't follow their "manageable expression" directly,
 because their equation seems wrong (7/30/2020). Insteadly it follows definition 
 above the "manageable expression" directly, which is also described in above doc.
 """
-struct CorrBijector <: Bijector{2} end
+struct CorrBijector <: Bijector end
 
-function (b::CorrBijector)(x::AbstractMatrix{<:Real})    
+with_logabsdet_jacobian(b::CorrBijector, x) = transform(b, x), logabsdetjac(b, x)
+
+function transform(b::CorrBijector, x::AbstractMatrix{<:Real})
     w = cholesky(x).U  # keep LowerTriangular until here can avoid some computation
     r = _link_chol_lkj(w) 
     return r + zero(x) 
     # This dense format itself is required by a test, though I can't get the point.
     # https://github.com/TuringLang/Bijectors.jl/blob/b0aaa98f90958a167a0b86c8e8eca9b95502c42d/test/transform.jl#L67
 end
 
-(b::CorrBijector)(X::AbstractArray{<:AbstractMatrix{<:Real}}) = map(b, X)
-
-function (ib::Inverse{<:CorrBijector})(y::AbstractMatrix{<:Real})
+function transform(ib::Inverse{<:CorrBijector}, y::AbstractMatrix{<:Real})
```

Comment:
Since there are no type parameters anymore:
```suggestion
function transform(ib::Inverse{CorrBijector}, y::AbstractMatrix{<:Real})
```